README.md

```diff
@@ -1,86 +1,86 @@
-            </tr>
-            <tr>
-                <td align="left">authorizationUrl</td>
-                <td align="left">Url of the 'GetDetails' action in the ASP.NET application.</td>
-            </tr>
-        </tbody>
-        </table>
-
-### Install npm
-
-To install all dependent packages, use the below command
-
-```bash
-npm install
-```
-### Run/Serve
-
-To run the samples, use the below command
-
-```bash
-node embed.js
-```
-
-Please refer to the [help documentation](https://help.boldbi.com/embedded-bi/javascript-based/samples/v3.3.40-or-later/node-js/#how-to-run-the-sample) to know how to run the sample.
-
-## Online Demos
+  * Google Chrome, Microsoft Edge, Mozilla Firefox, and Safari.
+
+ ## Configuration
+
+ * Please ensure that you have enabled embed authentication on the `embed settings` page. If it is not currently enabled, please refer to the following image or detailed [instructions](https://help.boldbi.com/site-administration/embed-settings/#get-embed-secret-code) to enable it.
+
+   ![Embed Settings](https://github.com/boldbi/aspnet-core-sample/assets/91586758/b3a81978-9eb4-42b2-92bb-d1e2735ab007)
+
+ * To download the `embedConfig.json` file, please follow this [link](https://help.boldbi.com/site-administration/embed-settings/#get-embed-configuration-file) for reference. Additionally, you can refer to the following image for visual guidance.
+
+    ![Embed Settings Download](https://github.com/boldbi/aspnet-core-sample/assets/91586758/d27d4cfc-6a3e-4c34-975e-f5f22dea6172)
+    ![EmbedConfig Properties](https://github.com/boldbi/aspnet-core-sample/assets/91586758/d6ce925a-0d4c-45d2-817e-24d6d59e0d63)
+
+ * Copy the downloaded embedConfig.json file and paste it into the designated [location](https://github.com/boldbi/nodejs-sample) within the application. Please ensure that you have placed it in the application as shown in the following image.
+
+    ![EmbedConfig image](https://github.com/boldbi/aspnet-core-sample/assets/91586758/5475fe01-2f43-4388-b91d-c6447f101aa4)
+
+ ## Run a Sample Using Command Prompt
+    
+  1. Open the command prompt and navigate to the specified file [location](https://github.com/boldbi/nodejs-sample) where the project is located.
+
+  2. To install all dependent packages, use the below command
+
+     ```bash
+     npm install
+     ```
+  
+  3. Finally, run the application using the below command
+ 
+     ```bash
+     node embed.js
+     ```
+
+ ## Developer IDE
+
+  * Visual studio code(https://code.visualstudio.com/download)
+
+  ### Run a Sample Using Visual Studio Code
+ 
+  * Open the NodeJS sample in Visual Studio Code. 
+   
+  * Open the terminal in Visual Studio Code and install all dependent packages by executing the following command in the terminal
+    ```bash
+    npm install
+    ```
+ 
+  * Finally, run the application using the below command. 
```

Comment:
* Finally, run the application using the following command.

Reply:
Content updated